tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/OutputMode.scala

```diff
@@ -36,33 +36,38 @@ object OutputMode {
     val esLevel: ESLevel = ESLevel.ES5
   }
 
-  /** Modern output mode compliant with ECMAScript 5.1 in a function scope.
-   *  This is the default output mode used by fastOpt and fullOpt.
-   *  The output must be enclosed in an anonymous function isolating the code
-   *  in a dedicated scope.
+  /** Output mode compliant with ECMAScript 5.1 (deprecated alias).
+   *
+   *  This value is not annotated with `@deprecated` for technical reasons, but
```

Comment:
What are the "technical reasons"? Please document them (internally is fine). Also, how are users supposed to notice that we deprecated this?